VERSION.txt

```diff
@@ -1 +1 @@
-0.2.0
+0.3.0
```

Comment:
In our current release process, the version number only gets bumped during the release cycle, not during individual PRs (so we can batch multiple changes in 1 version bump). We are in the process of updating our release process, but for now, could you remove the corresponding commit and `push -f`?